README.markdown

```diff
@@ -1,23 +1,36 @@
 # Puppet Docs
 
-Curated documentation for Puppet.
+Curated documentation and issue tracker for Puppet.
 
 ## Where is everything?
 
 Most of our documentation has been moved out of this repository. This repo contains:
   * Some of our older, unmaintained documentation, which is deprecated and may be removed from this repo without further notice. For archived docs for unmaintained, unsupported versions of Puppet, see the [Puppet docs archive](https://github.com/puppetlabs/docs-archive).
   * Reference documentation generated from code. This reference documentation is then rendered into HTML and included in the Puppet docs website.
+  * The issue tracker allowing you to suggest improvements to puppet.com/docs
 
 As of Puppet 5.5, Puppet docs were migrated into the DITA XML format and have been maintained in a CMS. We removed each migrated page from this repository, so that the old content wasn't accidentally published.
 
 As of Facter 3.12, the Facter docs were migrated to DITA and organized as part of the Puppet doc set. Updates to 3.12 (and later) should be made in DITA, not in this repo. The generated `core_facts.md` file is still this repository, but has moved to the `puppet/x.y` folder instead of `facter/x.y`.
 
-## Contributions from the Puppet community are welcome
+## Reporting Issues
 
-If something in the documentation doesn't seem right, or if you think something important is missing, we want to hear from you! Please send us feedback in any of the following ways:
-* Use the feedback form on the relevant page to send us details of the issue.
-* Open a Jira ticket in the DOCUMENT project here: https://tickets.puppetlabs.com/projects/DOCUMENT/ Let us know the URL of the page, and describe the changes you think it needs.
-* Email `docs@puppet.com` if you have questions about contributing to the documentation.
+This repository will allow public community members to suggest documentation improvements.
```

Comment:
Line 18: Let's change the text to focus on the user:
"Public community members can use this repository to suggest documentation improvements."
Line 19: I would change "Puppet TechPubs" to "Technical Publications Team."
Line 20: I would change "may" to "might."
Line 22: I would add a comma after "in this way."
Line 24: Let's remove "please." "Please" is understood differently in different cultures.
Line 26: Let's add a period at the end.
Line 27: Let's change the number 1 to 2, remove the word "through," and replace the period with a colon.
Line 28: Let's remove the word "then."
Line 30: Let's remove the word "then."
Line 31: Let's change the text to say:

"Include the full URL of the documentation webpage for which you are requesting updates."